people_recognition_3d/scripts/people_recognition_3d_node

```diff
@@ -28,17 +28,24 @@ class PeopleRecognition3DNode:
         padding = self._get_param('~padding', 5)
 
         enable_topic_mode = self._get_param('~enable_topic_mode', False)
+        depth_info_sub_name = self._get_param('~camera_info_depth', 'camera_info')
```

Comment:
There is no need for using ros parameters here, that is what remapping is for. 
See https://wiki.ros.org/Remapping%20Arguments and https://wiki.ros.org/roslaunch/XML/remap

Reply:
Its done.